web/locales/fi.json

```diff
@@ -0,0 +1,1786 @@
+        "renewableLowCarbonDifference-question": "Mitä eroa on 'uusiutuvilla' ja 'vähähiilisillä' energianlähteillä?",
+        "renewableLowCarbonDifference-answer": "Uusiutuvan energiantuotanto perustuu uusiutuviin energialähteisiin, kuten tuulen ja veden virtaukseen, auringonpaisteeseen ja maalämpöön. Vähähiilinen energiantuotanto tarkoittaa, että tuotantoon liittyy erittäin alhainen kasvihuonekaasupäästö, esimerkiksi ydinvoiman tuotannossa.",
+        "importsAndExports-question": "Otattekoko huomioon sähkön tuonnin ja viennin?",
+        "importsAndExports-answer": "Kyllä otamme. Tuonnit ja viennit voi nähdä kartalla pieninä <a href=\"#mapArrows\" class=\"entry-link\">nuolina alueiden välillä</a>. Yksityiskohtaiset tiedot näkyvät taulukoissa, kun klikkaat aluetta.",
+        "emissionsOfStored-question": "Entä päästöt, jotka syntyvät sähköntuotannosta, joka sitten varastoidaan paristoihin tai käytetään säiliöiden täyttämiseen?",
+        "emissionsOfStored-answer": "Koska tällä hetkellä vain pieni osa sähköstä on varastoitu, näiden päästöjen mallintamisen epätarkkuuksien ei pitäisi olla merkittävä ero nykyisen kokonaismäärän kanssa. Varastoinnin kasvavan merkityksen vuoksi toivomme kuitenkin voivan ottaa se paremmin huomioon mallissamme pian.",
+        "guaranteesOfOrigin-question": "Mitä ovat vihreät todistukset ja kuinka ne otetaan huomioon?",
+        "guaranteesOfOrigin-answer": "Kun uusiutuvan energian tuottajat tuottavat sähköä, he voivat luoda alkuperätakuut (tai uusiutuvan energian todistukset) - todisteen siitä, että uusiutuvaa sähköä on tuotettu ja jaettu sähköverkkoon. Nämä takuut voidaan sitten myydä, jolloin sähkön vähittäiskauppiaat voivat väittää, että heidän jälleenmyyntinsä tulee uusiutuvista lähteistä riippumatta sen todellisesta fyysisestä lähteestä. <br> <br> Tämä tarkoittaa, että sähkön kuluttajille voidaan kertoa, että heidän sähkönsä tulee aurinko- ja tuulivoimasta, jopa tuulettomana yönä. Tämä on ongelmallista, koska se poistaa kuluttajien kannustimet kuluttaa sähköä parhaalla mahdollisella hetkellä (esimerkiksi kun uusiutuvan sähkön määrä verkossa on tosiasiallisesti suurin). Tästä syystä tämä kartta sulkee pois alkuperätakuut ja sen sijaan tarjoaa sijainnin perusteella oikean kuvan sähköverkosta kuluttajien käyttöön.",
+        "otherSources-question": "Miksi ette näytä muita päästölähteitä kuin sähköntuotannon?",
+        "otherSources-answer": "Tämä olis projekting laajuden ulkopuolella. Tomorrow pyrkii kuitenkin aktiivisesti kehittäämaan <a href=\"#whoAreYou\" class=\"entry-link\"> uusia ratkaisuja arjen ilmastovaikutusten määrittämiseksi </a>.",
```

Comment:
Bunch of typos:
`olis` => `olisi`
`projekting` => `projektin`
`kehittäämaan` => `kehittämään`